web/components/OnboardingSlide.tsx

```diff
@@ -0,0 +1,161 @@
+import {
+  Flex,
+  Text,
+  Tooltip,
+  Stack,
+  Modal,
+  ModalOverlay,
+  ModalContent,
+  ModalBody,
+  IconButton,
+  useDisclosure,
+  ModalCloseButton,
+} from "@chakra-ui/react";
+import { QuestionOutlineIcon } from "@chakra-ui/icons";
+import OnboardingOptions from "./OnboardingOptions";
+
+enum QType {
+  Intro,
+  Question,
+  Completion,
+}
+
+function OnboardingSlide(props: {
+  questionData: {
+    title: string;
+    description: string;
+    options: string[];
+    qtype: QType;
+    singleAnswer: boolean;
+    dropdown: boolean;
+    tooltip: string;
+  }[];
```

Comment:
Define these externally since they're reused often - if some components don't use certain fields, you can have them use Typescript utility types (https://www.typescriptlang.org/docs/handbook/utility-types.html). Take a look at Pick, Omit, Exclude, etc...